Modules/Core/ImageFunction/include/itkWindowedSincInterpolateImageFunction.h

```diff
@@ -141,10 +141,10 @@ class ITK_TEMPLATE_EXPORT BlackmanWindowFunction
 
 private:
   /** Equal to \f$ \frac{\pi}{m} \f$ */
-  static const double m_Factor1;
+  static constexpr double m_Factor1 = itk::Math::pi / VRadius;
 
   /** Equal to \f$ \frac{2 \pi}{m} \f$  */
-  static const double m_Factor2;
+  static constexpr double m_Factor2 = 2.0 * itk::Math::pi / VRadius;
```

Comment:
Same here: your proposed change is OK to me, but I think these factors should be made local to the corresponding `operator()`.

Reply:
done.